main.go

```diff
@@ -58,6 +99,14 @@ func main() {
 		tmpls[filepath.Base(file)] = template.Must(template.Must(layouts.Clone()).ParseFiles(file))
 	}
 
+	if env.Get("TRACING_ENABLED", "0") == "1" {
+		shutdown := initTracerProvider(context.Background(), logger)
+		defer shutdown()
+	}
+
+	httpClient := http.DefaultClient
+	httpClient.Transport = otelhttp.NewTransport(httpClient.Transport)
+
 	client, err := sirius.NewClient(http.DefaultClient, siriusURL)
```

Comment:
```suggestion
	client, err := sirius.NewClient(httpClient, siriusURL)
```
Otherwise you've created `httpClient` for nothing :sob:

Reply:
Whoops 😓